suite-native/module-settings/src/tests/utils.test.ts

```diff
@@ -0,0 +1,7 @@
+import { transformFiatCurrencyToSelectItem } from '../utils';
+
+test('transformFiatCurrencyToSelectItem returns correct value', () => {
+    expect(
+        transformFiatCurrencyToSelectItem({ label: 'usd', value: 'United States Dollar' }),
+    ).toEqual({ value: 'usd', label: 'United States Dollar' });
```

Comment:
Why is the util param and the expected result equal? Am I missing something?

Reply:
Value and label properties swap here.

Reply:
Ah, pardon 😄